sys/net/gnrc/sock/udp/gnrc_sock_udp.c

```diff
@@ -221,11 +221,18 @@ ssize_t sock_udp_send(sock_udp_t *sock, const void *data, size_t len,
         if (sock != NULL) {
             /* bind sock object implicitly */
             sock->local.port = src_port;
+            if (remote == NULL) {
+                sock->local.family = sock->remote.family;
```

Comment:
Yes. The line I just added a minor bugfix above guarantees that either `remote != NULL` or `sock->remote.family != AF_UNSEC`.